tao_compiler/mlir/disc/tests/regression/data/kstitch_fusion_with_splat_const.mlir

```diff
@@ -0,0 +1,12 @@
+module attributes {tf.versions = {bad_consumers = [], min_consumer = 0 : i32, producer = 0 : i32}} {
+  func @main(%arg0: tensor<?x?xf32>) -> (tensor<?xf32>, tensor<?xf32>) attributes {tf.entry_function = {inputs = "{{INPUTS}}", outputs = "{{OUTPUTS}}", input_placements="{{INPUT_PLACEMENTS}}", output_placements="{{OUTPUT_PLACEMENTS}}"}} {
+    %graph:2 = tf_executor.graph {
+      %1:2 = tf_executor.island wraps "tf.Const"() {value = dense<[1]> : tensor<1xi32>} : () -> tensor<1xi32>
```

Comment:
Just curious about it. This const was not supported by previous fusion because the rank is 1, right? (Even it only contains one element.)

Reply:
rank 0 const was supported in previous fusion, rank n splat const (n > 0) was not supported.
in this case, due to the implicit broadcast and const fold, it's actually a rank n splat const.